code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -439,7 +568,135 @@
 
 /obj/structure/closet/proc/tool_interact(obj/item/W, mob/living/user)//returns TRUE if attackBy call shouldn't be continued (because tool was used/closet was of wrong type), FALSE if otherwise
```

Comment:
No single letter vars `W` needs to be renamed

Reply:
It happens